templates/admin/emails/list.tmpl

```diff
@@ -38,6 +38,7 @@
 						</th>
 						<th>{{ctx.Locale.Tr "admin.emails.primary"}}</th>
 						<th>{{ctx.Locale.Tr "admin.emails.activated"}}</th>
+						<th></th>
```

Comment:
How about
```suggestion
						<th>{{ctx.Locale.Tr "admin.notices.op"}}</th>
```

Although I don't think it's a good idea to reuse the word which belong to `admin.notices`, but it do be reused by admin repo list and admin package list. Whatever, it would be better than leaving it empty.

Reply:
Emmm, it is not necessary maybe. As other pages also don't have this. (But some pages do)
e.g.
![QQ_1721787456195](https://github.com/user-attachments/assets/49c59427-5aaa-4f4c-bf57-60d53875ff09)